proto/rill/local/v1/api.proto

```diff
@@ -42,3 +52,44 @@ message GetVersionResponse {
   string current = 1;
   string latest = 2;
 }
+
+message DeployValidationRequest {}
+
+message DeployValidationResponse {
+  bool is_authenticated = 1; // if true below fields are relevant after login
+  string login_url = 2; // redirect to this if is_authenticated is false
+  bool is_github_connected = 3; // if true below fields are relevant after github install
+  string github_grant_access_url = 4; // redirect to this if is_github_connected or is_github_repo_access_granted is false
+  string github_user_name = 5;
+  rill.admin.v1.GithubPermission github_app_user_permission = 6; // if unspecified then github app not installed on user account
+  map<string, rill.admin.v1.GithubPermission> github_organization_installation_permissions = 7;
```

Comment:
Is there a reason to use "app" in the first name and "installation" in the second? Otherwise, could consider making the names more similar, e.g. `github_user_permissions` and `github_organization_permissions`.